test/integration-tests/sdk-test-suites/cpp/function_provider.cpp

```diff
@@ -38,12 +38,12 @@ class FunctionProvider : public FunctionTalent {
 static auto mqtt_config = json{
     {"platform", true},
     {"module", {
-                   {"name", "./testset_sdk/cpp/build/iotea-sdk-cpp-lib/adapters/mqtt/libmqtt_protocol_adapter.so"}
+                   {"name", "./iotea-sdk-cpp-lib/adapters/mqtt/libmqtt_protocol_adapter.so"}
```

Comment:
Same as above: The configurations of the other test-suites are in separate config.json files located in test/integration-tests/config/tests/<language>. It would be good to have the same structure for cpp.

Reply:
Good point. There are other optimizations possible but this makes sense to provide flexible configurations.
I'll change

Reply:
Done :)